### DISCUSSION TITLE
Codice JSON Garage per sonoff mini

### DISCUSSION THREAD

**Discussion by anto-toto** [RavenSystem/esp-homekit-devices#1773] [MEPLHAA Scripts]
Buonasera a tutti, Ho un sonoff mini e vorrei utilizzarlo per comandare il garage. Ho bisogno che il sonoff apra il garage con un impulso in quanto l’ho collegato al pulsante di apertura. Dopo aver dato l’impulso di apertura ho bisogno che il garage si fermi a una certa altezza in quanto la serranda é molto alta e non vorrei aprirla tutta. Quindi, dopo l’impulso di apertura, in automatico vorrei che il sonoff inviasse un altro impulso per bloccare la salita della serranda ad esempio dopo 20 secondi dall’apertura. E ovviamente con l’impulso di chiusura il sonoff deve solo dare un impulso per far scendere la serranda.

**Comment by anto-toto** [comment]
Good evening everyone, I have a sonoff mini and I would like to use it to control the garage. I need the sonoff to open the garage with a pulse as I connected it to the opening button. After giving the opening impulse I need the garage to stop at a certain height as the shutter is very high and I wouldn't want to open it all. So, after the opening pulse, I would automatically like the sonoff to send another pulse to block the ascent of the shutter for example after 20 seconds of opening. And of course with the closing impulse the sonoff just has to give an impulse to bring down the shutter.

0

1

**Comment by branda92** [comment]
Ciao, che tipo di garage hai? A basculante o avvolgibile?

**Comment by anto-toto** [comment]
Ciao,
Il garage è Avvolgibile

Saluti
Antonio Vitrò
